src/Service/Security/PackageScanner/SensioLabsPackageScanner.php

```diff
@@ -144,10 +151,6 @@ private function extractLockFiles(string $distFilename): array
 
         $zip->close();
 
-        if ($lockFiles === []) {
-            throw new \RuntimeException('Lock file not found');
-        }
```

Comment:
:thinking: what about new status? see https://github.com/repman-io/repman/issues/189#issuecomment-637975628

Reply:
You're right, I've added `n/a` status